### ISSUE TITLE
Sourcetrail gets SIGSEGV when quitting the program

### ISSUE THREAD

**Issue by danielcwilliams** [CoatiSoftware/SourcetrailBugTracker#698]
To reproduce: start the program and then immediately type Ctrl-Q

Sourcetrail Version 2019.2.25, software is commercially licensed, not installed to /opt but run from a subdirectory of $HOME

OS is Ubuntu 18.04.2 LTS

  $ uname -a
  Linux cvdev2 4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

traceback in the core file:

```
Program terminated with signal SIGSEGV, Segmentation fault.
(gdb) where
#0  __memset_avx2_unaligned () at ../sysdeps/x86_64/multiarch/memset-vec-unaligned-erms.S:116
#1  0x00000000029c31c9 in Botan::HMAC_DRBG::~HMAC_DRBG() ()
#2  0x00000000029c16ea in Botan::AutoSeeded_RNG::~AutoSeeded_RNG() ()
#3  0x0000000000ef2acf in License::~License() ()
#4  0x0000000000ef4ef1 in std::unique_ptr<License, std::default_delete<License> >::~unique_ptr() ()
#5  0x00007f1f2eb85041 in __run_exit_handlers (status=0, listp=0x7f1f2ef2d718 <__exit_funcs>, run_list_atexit=run_list_atexit@entry=true, run_dtors=run_dtors@entry=true) at exit.c:108
#6  0x00007f1f2eb8513a in __GI_exit (status=<optimized out>) at exit.c:139
#7  0x00007f1f2eb63b9e in __libc_start_main (main=0xa06670 <main>, argc=1, argv=0x7fffb35355b8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>,
    stack_end=0x7fffb35355a8) at ../csu/libc-start.c:344
#8  0x0000000000a38951 in _start ()```

**Comment by egraether** [comment]
Thanks for reporting!

**Comment by egraether** [comment]
Fixed with 2019.2.27
We will release a new build soon, since this crash is affecting lots of users.
